code/game/objects/effects/spawners/random/random_spawner.dm

```diff
@@ -35,6 +35,8 @@
 	var/spawn_random_angle = FALSE
 	/// Whether blackbox should record when the spawner spawns.
 	var/record_spawn = FALSE
+	/// Where do we want to spawn an item (closet, safe etc.)
+	var/obj/spawn_inside = null
```

Comment:
This is just a typepath so we don't cast it, and vars are initialized to `null` by default so it's not necessary to specify it

```suggestion
	var/spawn_inside
```